### DISCUSSION TITLE
Week 2:Participatory Mapping

### DISCUSSION THREAD

**Discussion by gu-ana** [albertkun/21S-ASIAAM-191A#84] [Past Thinking Caps]
My thoughts on Participatory Mapping and Critical Cartography

I was really fascinated by Mariah's talk as I had always thought of maps to be clear cut and didn't explore it in ways that is important, which is to deliver information to the viewer. I really like the idea of Participatory Mapping as it gives voice to marginalized communities or communities that data tend to ignore to size or absence of data. Whether they construct their maps using GIS or their own traditional mapping, these communities have a much better idea of what they're presenting. 

There are drawbacks of participatory mapping that I am unsure how it can be resolved. At the end of the day, maps are used to communicate information to the person observing, but there are nuances that are going to be ignored or not understood by people not from that culture. Language already is a big indicator in how our brains perceive the world and it definitely affects how anybody interprets a piece of text or a medium such as a map. 

I never really thought about social mapping as a mapping but the examples given in next week's reading[1] reminded me of how a network of people are considered a "map"/graph in computer science. But I'm grateful that this class has opened my eyes.

https://onlinelibrary.wiley.com/doi/epdf/10.1002/j.1681-4835.2006.tb00163.x

**Comment by latishasumardy** [comment]
I agree! Participatory mapping definitely a very effective tool for marginalized communities, giving them the power to showcase important information regarding their communities that may have been suppressed or hidden. It also becomes much more powerful in the hands of people who actually understand the data that they are handling. I also agree about the drawbacks that you mentioned, and I believe that the only way to truly resolve these types of issues is to educate the public so that everyone will have the tools needed to understand what these maps are communicating.

**Comment by okcirE** [comment]
Agreed, I also enjoyed Mariah's lecture and that she introduced many of us to new sets of lenses when analyzing all types of maps. It would be beneficial for GIS programmers and map cartographers to learn and understand the culture behind the area that they are trying to map. In this way, we can at least portray a map that is not ignorant of the histories and rich culture of the area. I think the process of naming streets can also have this implementation.
